ApiReview/API.Polly.RateLimiting/NoDocs/Polly/RateLimiterCompositeStrategyBuilderExtensions.cs

```diff
@@ -0,0 +1,17 @@
+// Assembly 'Polly.RateLimiting'
+
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.RateLimiting;
+using Polly.RateLimiting;
+
+namespace Polly;
+
+public static class RateLimiterCompositeStrategyBuilderExtensions
+{
+    public static TBuilder AddConcurrencyLimiter<TBuilder>(this TBuilder builder, int permitLimit, int queueLimit = 0) where TBuilder : CompositeStrategyBuilderBase;
+    public static TBuilder AddConcurrencyLimiter<TBuilder>(this TBuilder builder, ConcurrencyLimiterOptions options) where TBuilder : CompositeStrategyBuilderBase;
+    public static TBuilder AddRateLimiter<TBuilder>(this TBuilder builder, RateLimiter limiter) where TBuilder : CompositeStrategyBuilderBase;
```

Comment:
this implementation requires the RateLimited to be constructed upfront. can it be changed to be lazy?

Reply:
I can be initialized lazily if created using `AddResilienceStrategy`.